### PR TITLE
Update files

### PR DESCRIPTION
**Pull request by ITeMbI4**
## Description
**Describe the changes** 
Update project credits

**What is the current behavior?** (You can also link to an open issue here)
-

**What is the new behavior?** (if this is a feature change)
-

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:

<br />

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [x] I have checked the project can be compiled
- [x] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Review by louis1706 — COMMENTED** [review]
for YamlDotNet i prefer to use the one from base game

**Comment by ITeMbI4** [comment]
There is no package with version 11.1.3.0, so I'll revert yaml changes

**Comment by Misaka-ZeroTwo** [comment]
image

Actually there is

**Comment by ITeMbI4** [comment]
Weird naming =)

**Comment by louis1706** [comment]
> Weird naming =)

still better to use the on from SCP:SL file directly so we don't need to update it

File: EXILED/Exiled.Loader/Exiled.Loader.csproj [hunk 1]
```diff
@@ -31,9 +31,9 @@
         <Reference Include="UnityEngine.CoreModule" HintPath="$(EXILED_REFERENCES)\UnityEngine.CoreModule.dll" Private="false" />
         <Reference Include="Mirror" HintPath="$(EXILED_REFERENCES)\Mirror.dll" Private="false" />
         <Reference Include="System.ComponentModel.DataAnnotations" Private="false" />
-        <Reference Include="YamlDotNet" HintPath="$(EXILED_REFERENCES)\YamlDotNet.dll" Private="false" />
         <Reference Include="System.Net.Http" />
         <Reference Include="Mono.Posix" HintPath="$(EXILED_REFERENCES)\Mono.Posix.dll" Private="false" />
+        <Reference Include="YamlDotNet" HintPath="$(EXILED_REFERENCES)\YamlDotNet.dll" Private="false" />
```
Comment (IRacle1) [inline EXILED/Exiled.Loader/Exiled.Loader.csproj:36]: 🤨

**Review by IRacle1 — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

**Pull request merged**